OutSystems/outsystems-ui
# ROU-4527 - [OSUI] - Accessibility improvements identified on team audit for the Rating

This PR is for improve rating component in terms of accessibility.

### What was done

- add logic to manage aria-hidden and area checked;
- remove the logic that was adding wcaghideText to the first input label;
- improve scss file;
- add styles to prevent half from being visible when is not half rating;
- add styles to guarantee the empty editable rating is visible;

### Test Steps

1. Open the test page
2. Run an A11Y checker
3. Run the page using a screen reader
4. Expected: No warnings related to rating

### Screenshots

(prefer animated gif)

### Checklist
